jake/test_parse.py

```diff
@@ -24,9 +24,9 @@ def setUp(self):
 
     def test_callGetDependenciesReturnsPurls(self):
         fn = pathlib.Path(__file__).parent / "condalistoutput.txt"
-        sys.stdin = open(fn, "r")
-        actual = self.func.getDependenciesFromStdin(sys.stdin)
-        output = json.loads(actual)["coordinates"]
+        with open(fn, "r") as stdin:
+            actual = self.func.getDependenciesFromStdin(stdin)
+            output = actual.get_coordinates()
```

Comment:
I probably just don't understand Python well enough, but this leads me to believe output would die at the end of the with block?

Reply:
Now I get it, it only closes the `stdin` and for some reason the indented variables are not block scoped, GO FIGURE!